src/components/date-input/template.test.js

```diff
@@ -459,4 +459,25 @@ describe('Date input', () => {
     expect($monthInput.hasClass('undefined')).toBeFalsy()
     expect($yearInput.hasClass('undefined')).toBeFalsy()
   })
+
+  describe('when it includes autocomplete attributes', () => {
+    it('renders the autocomplete attribute', () => {
+      const $ = render('date-input', {
+        items: [
+          {
+            'autocomplete': 'bday-day'
+          },
+          {
+            'autocomplete': 'bday-month'
+          },
+          {
+            'autocomplete': 'bday-year'
+          }
+        ]
+      })
+
+      const $firstItems = $('.govuk-date-input__item:first-child input')
+      expect($firstItems.attr('autocomplete')).toEqual('bday-day')
```

Comment:
Non-blocking: I wonder if we should assert that all three are there...

Reply:
Thanks Nick 👍 Going to leave it as it is for now to keep things consistent, there's about 10 tests in this file that follow the same pattern. Maybe it's something we can look at separately.